src/components/legends/unmarkedWinterStorageLegend/UnmarkedWinterStorageLegend.tsx

```diff
@@ -19,11 +23,7 @@ const UnmarkedWinterStorageLegend = ({ legend, form, steps }: UnmarkedWinterStor
       <Container>
         <Row>
           <Col lg={{ size: 10, offset: 1 }} xl={{ size: 8, offset: 2 }}>
-            {steps && (
-              <div className="vene-unmarked-winter-storage-legend__steps">
-                <Steps steps={steps} />
-              </div>
-            )}
+            {steps && <Steps steps={steps} />}
```

Comment:
Do we need these defensive checks for `steps`, `legend` and `form` ? seems that they are always given?

Reply:
Good point, removed